todo-list/src/components/ListItem/ListItem.scss

```
@@ -0,0 +1,116 @@
+@import "./../../styles/variables.scss";
+
+.list-item {
+  display: flex;
+  list-style-type: none;
+  margin-left: 0;
+  padding: 15px;
+  position: relative;
+  border-bottom: 1px solid $color-list-border;
+  transition-duration: $tranistion-duration;
+
+  &:last-of-type {
+    border-bottom: none;
+  }
+
+  .list-item-icon {
+    width: $icon-width;
+    position: relative;
+    display: flex;
+    align-items: center;
+
+    &:before {
+      content: "";
+      opacity: 0;
+      transition-duration: $tranistion-duration;
+      position: absolute;
+      left: 0;
+      right: 0;
+      margin: 0 auto;
+      color: $color-font-main;
+      border-color: $color-font-main;
+    }
+  }
+
+  .list-item-content-wrapper {
+    width: calc(100% - #{$icon-width});
+    display: flex;
+    padding-left: 15px;
+
+    @media screen and (max-width: $bp-mobile) {
+      flex-direction: column;
+    }
+  }
+
+  .list-item-text {
+    width: 40%;
+    display: flex;
+    align-items: center;
+
+    @media screen and (max-width: $bp-mobile) {
```

If mobile-first principle applies, then the CSS rules for mobile views should be in the first place, then:

```suggestion
    @media screen and (min-width: $bp-mobile) {
```